rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -402,3 +402,19 @@ def image_slice(*inputs, file_root='', path='', annotations_file='', shard_id=0,
         image_decoder_slice = b.fusedDecoderCropShard(
             Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (image_decoder_slice)
+
+def audio(*inputs, file_root='', file_list_path='', bytes_per_sample_hint=[0], shard_id=0, num_shards=1, random_shuffle=False, downmix=False, dtype=types.FLOAT, quality=50.0, max_frames=1 , max_channels=1 ,sample_rate=0.0, seed=1, storage_type=0, stick_to_shard=False, shard_size=-1):
```

Comment:
Please remove the extra space after max_frames=1, max_channels=1
and add space after max_channels=1 ,